tests/docker/python.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+ARG PRIVATE_KEY_FILE
+ARG CLIENT_ID
+ARG SCOPES
+COPY ${PRIVATE_KEY_FILE} /home/app/src/private_key.pem
```

Comment:
Can we avoid putting the private key into the docker cache? One option is to mount it dynamically with the docker -v option under the same filename on all test containers which would make PRIVATE_KEY_FILE variable unnecessary as well.